types/kendo-ui/index.d.ts

```diff
@@ -1848,7 +1848,7 @@ declare namespace kendo.ui {
         dates?: any;
         depth?: string;
         disableDates?: any|Function;
-        footer?: string|Function;
+        footer?: boolean|string|Function;
```

Comment:
Might as well make the type `false | string | Function` so that `true` isn't allowed accidentally.